app/assets/stylesheets/components/_places-search.scss

```diff
@@ -0,0 +1,21 @@
+.places-search {
+  padding: 0 0 ($base-spacing * 2) 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `0 0 $base-spacing * 2` instead of `0 0 $base-spacing * 2 0`